tests/k8s/1-032_validate_dynamic_scaling/02-add-cluster-secrets.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cluster-1
+  labels:
+    "argocd.argoproj.io/secret-type": "cluster"
+stringData:
+  name: mycluster-1.com
+  server: https://mycluster-1.com
+type: Opaque
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cluster-2
+  labels:
+    "argocd.argoproj.io/secret-type": "cluster"
+type: Opaque
+stringData:
+  name: mycluster-2.com
+  server: https://mycluster-2.com
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: sleep 30s
+---
+# Retrigger reconcilation of ArgoCD
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd
+  labels:
+    foo: bar
+spec:
+  controller:
+    sharding:
+      dynamicShardingEnabled: true
+      minShards: 1
+      maxShards: 4
+      clustersPerShard: 1
```

Comment:
@ishitasequeira is this step necessary? ideally we should not have to do anything to trigger reconciliation, the operator should automatically notice that these secrets are created and that autoscaling must be done 

do we need to add a watch for secrets with this specific cluster label maybe?

Reply:
The watcher is already added on Secrets. Removing the re-triggering of reconcilation